src/web_interface/templates/base.html

```diff
@@ -104,123 +108,179 @@
+        Firmware Analysis and Comparison Tool
+    </div>
+    <div id="navbarCollapse" class="collapse navbar-collapse">
+        <!-- Navbar Elements -->
+        <ul class="navbar-nav" style="padding-right: 5px">
+            {% for caption, href, icon in navigation_bar %}
+            {% if caption == "Database" %}
+            <li class="dropdown pr-3">
+                <a {% if caption== active_page %}style="color: #fff; font-weight: 500;" {% endif %}
+                   class="dropdown-toggle" data-toggle="dropdown" href="#"><i
+                        class="fas fa-tasks"></i>&nbsp;Database<span class="caret"></span></a>
+                <div class="dropdown-menu">
+                    <a class="dropdown-item" href="/database/browse" style="font-size: 14px"><i
+                            class="fas fa-list-ul"></i> Browse Firmware</a>
+                    <a class="dropdown-item" href="/database/browse_compare" style="font-size: 14px"><i
+                            class="far fa-copy"></i> Browse Compares</a>
+                    <a class="dropdown-item" href="/database/search" style="font-size: 14px"><i
+                            class="fas fa-search"></i> Basic Search</a>
+                    <a class="dropdown-item" href="/database/advanced_search" style="font-size: 14px"><i
+                            class="fas fa-search-plus"></i> Advanced Search</a>
+                    <a class="dropdown-item" href="/database/binary_search" style="font-size: 14px"><i
+                            class="fas fa-search-dollar"></i> Binary Pattern Search</a>
+                </div>
+            </li>
+            {% elif caption == "Info" %}
+            <li class="dropdown pr-3">
+                <a {% if caption== active_page %}style="color: #fff; font-weight: 500;" {% endif %}
+                   class="dropdown-toggle" data-toggle="dropdown" href="#"><i class="fas fa-info-circle"></i>&nbsp;{{
+                    caption }}<span class="caret"></span></a>
+                <div class="dropdown-menu">
+                    <a class="dropdown-item" href="/statistic" style="font-size: 14px"><i class="fas fa-chart-bar"></i>
+                        Statistics</a>
+                    <a class="dropdown-item" href="/system_health" style="font-size: 14px"><i
+                            class="fas fa-heartbeat"></i> System</a>
+                    <a class="dropdown-item" href="/about" style="font-size: 14px"><i class="fas fa-beer"></i> About</a>
+                </div>
+            </li>
+            {% elif caption == "Feedback" %}
+            <li class="pr-3">
+                <a {% if caption== active_page %}style="color: #fff; font-weight: 500;" {% endif %}
+                    data-toggle="modal" data-target="#staticBackdrop"> <i class="fas fa-bullhorn"></i> Feedback </a>
+            </li>
+            <div class="modal fade" id="staticBackdrop" data-backdrop="static" data-keyboard="false" tabindex="-1"
```

Comment:
we could probably find a better name for this element than "staticBackdrop". How about "feedbackModal"?
Also this should be outside the `<ul>` Tag (at least PyCharm is complaining that `<div>`s aren't allowed here)